interfaces/api/v1/world/bible.py

```
@@ -906,28 +906,28 @@ async def _sse_bible_generator(
-                                parent_id=pd.get("parent_id"),
-                            )
-                            location_ids.append((pd["location_id"], pd))
-                        except Exception:
-                            pass
                 elif item["type"] == "chunk":
                     yield _sse_fmt("data", {
                         "type": "location_chunk",
                         "chunk": item["text"],
                     })
 
+            # 流式收集完毕后批量落库，保证 raw_to_final 映射完整，parent_id 层级正确
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)